README.md

```diff
@@ -15,12 +15,12 @@ as in previous versions of the *Transaction Service*.
 Transactions can also be sent to the service to allow offchain collecting of signatures or informing the owners about
 a transaction that is pending to be sent to the blockchain.
 
-[Swagger (Mainnet version)](https://safe-transaction.gnosis.io/)
-[Swagger (Rinkeby version)](https://safe-transaction.rinkeby.gnosis.io/)
+[Swagger (Mainnet version)](https://safe-transaction-mainnet.safe.global/)
+[Swagger (Göerli version)](https://safe-transaction-goerli.safe.global/)
 
 ## Index of contents
 
-- [Docs](https://docs.gnosis.io/safe/docs/services_transactions/)
+- [Docs]( https://docs.gnosis-safe.io/backend/service-architecture)
```

Comment:
@Uxio0 I didn't find the exactly documentation for ` service_transactions`, what do you think about this one?

Reply:
Works for me